Dockerfile.assisted-service

```diff
@@ -1,16 +1,30 @@
-ARG BASE_TAG=stream9
+ARG RHEL_VERSION=9
+
+FROM registry.access.redhat.com/ubi$RHEL_VERSION/go-toolset:1.20 AS golang
+
+ADD . /app
+WORKDIR /app
+
+# Licenses
+RUN go install github.com/google/go-licenses@v1.6.0
+RUN ${HOME}/go/bin/go-licenses save --save_path /tmp/licenses ./...
 
 # Build binaries
-FROM quay.io/centos/centos:$BASE_TAG as builder
+FROM quay.io/centos/centos:stream$RHEL_VERSION as builder
```

Comment:
why we don't build from the UBI?

Reply:
related to the subscription manager needed on UBI images.